include/constants.h

```diff
@@ -34,6 +34,8 @@ namespace reinforcement_learning {  namespace name {
       const char *const  QUEUE_MODE = "queue.mode";
       const char *const  INTERACTION_FILE_NAME = "interaction.file.name";
       const char *const  OBSERVATION_FILE_NAME = "observation.file.name";
+      const char *const  HTTP_CLIENT_DISABLE_CERT_VALIDATION  = "http.certvalidation";
```

Comment:
We should keep the negations of the constant name and the constant value similar. In other words, I would expect false to mean that we do cert validation when I read `HTTP_CLIENT_DISABLE_CERT_VALIDATION`, but the opposite when I read `http.certvalidation`.